doc/folders.md

```diff
@@ -320,6 +327,7 @@ Metadata can be created on a folder by calling
 Note: This method will only succeed if the provided metadata template is not currently applied to the folder, otherwise
 it will fail with a Conflict error.
 
+<!-- sample post_folders_id_metadata_id_id -->
```

Comment:
Similar to the files/metadata endpoints I'm not sure if you wanted to go by url?